DESCRIPTION

```diff
@@ -8,18 +9,22 @@ Description: Provides easier interaction with
     returns an R data frame.
     Converts dates to 'POSIX' format.
     Manages throttling by 'Socrata'.
-Version: 1.6.1-2
-Date: 2015-6-5
-URL: https://github.com/Chicago/RSocrata
-BugReports: https://github.com/Chicago/RSocrata/issues
-Imports:
-    httr (>= 0.3),
-    jsonlite (>= 0.9.14),
-    mime (>= 0.2),
+Version: 1.6.2
+Date: 2015-7-15
+Authors@R: c(
```

Comment:
We'll actually change this back to the old `Author` before pushing to CRAN. The CRAN guys are *really* picky that my email address in `Authors@R` match that of the `Maintainer`'s email address. However, I like to separate the emails for general contact information from contacting me for package maintenance (CRAN will send e-mail blasts and I also use the email address in other projects I maintain). I prefer the `Authors@R`, but was getting cross with the CRAN submission process so used the old style.